introduction.md

```diff
@@ -1,2 +1,5 @@
  hello everyone! atte: yo mera
  my favorite Code school path so far in HTML & CSS
+*git
+*java
+*ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?